src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -278,10 +278,13 @@ public enum X509ChainStatusFlags
         CtlNotTimeValid = 131072,
         CtlNotValidForUsage = 524288,
         Cyclic = 128,
+        ExplicitDistrust = 67108864,
         HasExcludedNameConstraint = 32768,
         HasNotDefinedNameConstraint = 8192,
         HasNotPermittedNameConstraint = 16384,
+        HasNotSupportedCriticalExtension = 134217728,
         HasNotSupportedNameConstraint = 4096,
+        HasWeakSignature = 1048576,
```

Comment:
@joshfree @terrajobst 

All these additions to the `ref` folder are public API changes. Since we never released this contract before, this isn't breaking. But does this need to go thru API review?  Or is this considered 'de minimis' ?